src/current/v24.2/comment-on.md

```diff
@@ -204,6 +249,50 @@ To remove a comment from a database:
 (4 rows)
 ~~~
 
+### Remove a comment from a type
+
+Suppose we [create a type]({% link {{ page.version.version }}/create-type.md %}):
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TYPE IF NOT EXISTS my_point AS (x FLOAT, y FLOAT, z FLOAT);
+~~~
+
+To view the type you just created, use [`SHOW TYPES`]({% link {{page.version.version}}/show-types.md %}):
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW TYPES;
+~~~
+
+~~~
+  schema |   name   | owner
+---------+----------+--------
+  public | my_point | root
+(1 row)
+~~~
+
+To view all comments on types, make a [selection query]({% link {{page.version.version}}/select-clause.md %}) against the `system.comments` table:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM system.comments;
+~~~
+
+~~~
+  type | object_id | sub_id | comment
+-------+-----------+--------+-----------
+     7 |       112 |      0 | 3D POINT
+(1 row)
+~~~
+
+To remove a comment from a type, use a statement like the following:
```

Comment:
I'm not sure everything above this line is needed for this example. It duplicates the "Add a comment to a type" content and kind of buries the relevant line. Usually I say something like `To remove the comment from the type you created in the [preceding example](#add-a-comment-to-a-type)` or just leave it implicit that all the examples on the page are connected.

Reply:
SGTM, based on this and your other suggestion re: this section it's now

> To remove the comment from the type you created in the [preceding example](#add-a-comment-to-a-type), add a `NULL` comment:
> ~~~ sql
> COMMENT ON TYPE my_point IS NULL;
> ~~~